sdk/keyvault/azure-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/Es512.java

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.security.keyvault.keys.cryptography;
+
+class Es512 extends Ecdsa {
+    public static final String ALGORITHM_NAME = "ES512";
+
+    @Override
+    public int getDigestLength() {
+        return 64;
+    }
+
+    @Override
+    public int getCoordLength() {
+        return 66;
```

Comment:
66 or 64?

Reply:
Will follow up with KV team to double check but 66 is working-fine/in-sync with the KV service for now.